framework/src/media/audio/audio_manager.c

```diff
@@ -1730,6 +1730,7 @@ audio_manager_result_t get_device_process_handler_message(int card_id, int devic
 		return AUDIO_MANAGER_INVALID_DEVICE;
 	}
 	clock_gettime(CLOCK_REALTIME, &st_time);
+	st_time.tv_sec += 1;
```

Comment:
Actually 1 sec is very huge, we'd better use micro second instead of tv_sec.
Also this could be added in KConfig, how do you think?

Reply:
This PR is a makeshift.
I think there are three options we can choose.
1. Replace mq_timedreceive with mq_receive
2. Add time parameter
3. your suggestion

Which option do you think is the best?

Reply:
Definitely your suggestion.